public/examples/ach.html

```diff
@@ -113,14 +122,6 @@
           return;
         }
 
-        let card;
-        try {
-          card = await initializeCard(payments);
-        } catch (e) {
-          console.error('Initializing Card failed', e);
-          return;
-        }
-
```

Comment:
Thanks for removing the card flow from this example.  It just muddles up the example as is.